vita3k/gui/src/gui.cpp

```diff
@@ -534,7 +534,7 @@ void get_user_apps_title(GuiState &gui, EmuEnvState &emuenv) {
 
 void get_sys_apps_title(GuiState &gui, EmuEnvState &emuenv) {
     gui.app_selector.sys_apps.clear();
-    const std::vector<std::string> sys_apps_list = { "NPXS10003", "NPXS10008", "NPXS10015", "NPXS10026" };
+    const std::array<std::string, 4> sys_apps_list = { "NPXS10003", "NPXS10008", "NPXS10015", "NPXS10026" };
```

Comment:
You can even make this constexpr.

Reply:
where is proleme with vector ?

Reply:
@Macdu, fix it https://github.com/Vita3K/Vita3K/pull/2097/commits/2fe6d689e144335f27e6fa669216875a388de3a5